pages/index.md

```diff
@@ -24,62 +24,62 @@ items:
   - Experiental learning
 imgLinksItems:
 - image:
-    src: "/images/logos/logo-us-bank.png"
+    src: "/images/logos/logo-us-bank.svg"
     alt: U.S. Bank
   href: https://www.usbank.com/
   title: U.S. Bank
 - image:
-    src: "/images/logos/logo-lbg.png"
+    src: "/images/logos/logo-lbg.svg"
```

Comment:
I notice that the LBG logo differs from the old one (in this SVG the horse is above the text, whereas in the old PNG they are side-by-side).

It still looks fine to me, but considering how many iterations these logos have been through, perhaps @wipro-zara or @zanecunningham should take a look and give their thumbs up (or not).

Reply:
sure thing, @wipro-zara @zanecunningham you might like to review this too sometime? I've stuck the horse back to the right of the text, that was just how it was in the asset I found

Reply:
They're scaled quite differently when on full screen on a large desktop compared to the original, no? Otherwise, the images look far more crisp. I like it.

Reply:
👍yup I'll adjust some of the scaling